src/com/facebook/buck/android/AdbHelper.java

```diff
@@ -902,4 +956,23 @@ public static String tryToExtractPackageNameFromManifest(
       throw new HumanReadableException("Could not extract package name from %s", pathToManifest);
     }
   }
+
+  public static String tryToExtractInstrumentationTestRunnerFromManifest(
+      InstallableApk androidBinaryRule,
+      ExecutionContext context) {
+    Path pathToManifest = androidBinaryRule.getManifestPath();
```

Comment:
Good point, I didn't think about it first, but it still works, I just tested it.